app/src/app/townhall/page.tsx

```
@@ -39,39 +50,61 @@ import type { Village, VillageAlliance } from "@/drizzle/schema";
+import type { FetchActiveWarsReturnType } from "@/server/api/routers/war";
+import type { GlobalMapData } from "@/libs/travel/types";
+import StatusBar from "@/layout/StatusBar";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+
+const Map = dynamic(() => import("@/layout/Map"), { ssr: false });
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)